### PR TITLE
feat: [ACI-980] allow pre configured credly organizations

### PR DESCRIPTION
**Pull request by andrii-hantkovskyi**

### CODE REVIEW

File: credentials/apps/badges/admin.py [hunk 1]
```diff
@@ -164,6 +168,38 @@ def sync_organization_badge_templates(self, request, queryset):
...
+            fields.append("api_key") if "api_key" not in fields else None
+        return fields
+    
+    def get_readonly_fields(self, request, obj=None):
+        readonly_fields = super().get_readonly_fields(request, obj)
+
+        if not obj:
+            return readonly_fields
+
+        if str(obj.uuid) in settings.BADGES_CONFIG["credly"].get("ORGANIZATIONS", {}).keys():
```
Comment (wowkalucky) [inline credentials/apps/badges/admin.py:200]: Repetitive >> `get_preconfigured_organozations()`

File: credentials/apps/badges/models.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ class CredlyOrganization(TimeStampedModel):
     """
 
     uuid = models.UUIDField(unique=True, help_text=_("Put your Credly Organization ID here."))
-    api_key = models.CharField(max_length=255, help_text=_("Credly API shared secret for Credly Organization."))
+    api_key = models.CharField(max_length=255, help_text=_("Credly API shared secret for Credly Organization."), blank=True)
```
Comment (wowkalucky) [inline credentials/apps/badges/models.py:43]: DB migration?

Comment (andrii-hantkovskyi) [reply credentials/apps/badges/models.py:43]: added

File: credentials/apps/badges/admin.py (hunk 1 above)
Comment (andrii-hantkovskyi) [reply credentials/apps/badges/admin.py]: extracted

**Review by wowkalucky — APPROVED** [review]

**Pull request merged**